### ISSUE TITLE
Post gallery metabar localization strings

### ISSUE THREAD

**Issue by aerohub** [LouisBarranqueiro/hexo-theme-tranquilpeak#426]
It would be great to make it localizable with localization strings:

**layout/_partial/post/gallery.ejs**: [line 13](https://github.com/LouisBarranqueiro/hexo-theme-tranquilpeak/blob/master/layout/_partial/post/gallery.ejs#L13)
```
<span><%= 'Gallery : ' + photos.length + ' images' %></span>
```

**Comment by LouisBarranqueiro** [comment]
Sure

**Comment by LouisBarranqueiro** [comment]
Fixed in v1.11.0
